fastapi/exceptions.py

```diff
@@ -13,8 +16,10 @@ def __init__(
 
 
 class RequestValidationError(ValidationError):
-    pass
+    def __init__(self, errors: Sequence[ErrorList]) -> None:
+        super().__init__(errors, Request)
 
 
 class WebSocketRequestValidationError(ValidationError):
-    pass
+    def __init__(self, errors: Sequence[ErrorList]) -> None:
+        super().__init__(errors, WebSocket)
```

Comment:
Same thing applies here as applies to Request above.

Reply:
Another thing we could do would be to create new (empty) types called `RequestDependencies` and `WebsocketDependencies` to pass to `ValidationError` to make it even more clear what is happening. (But I'm also fine keeping it like this.)